utils/limits.tsx

```diff
@@ -37,18 +37,18 @@ export const fallbackStarterLimits = {
     },
 };
 
-// A negative usage value means they are over the limit. This function simply tells you whether ANY LIMIT has been reached/surpassed.
-export function anyUsageDeltaValueIsNegative(deltas: CloudUsage) {
-    let foundANegative = false;
+// A positive usage value means they are over the limit. This function simply tells you whether ANY LIMIT has been reached/surpassed.
+export function anyUsageDeltaExceededLimit(deltas: CloudUsage) {
```

Comment:
10 protons were polled as to whether they like this name change. They all responded "I'm positive".